### DISCUSSION TITLE
Unable to Open Monaco-editor in Theia-cloud deployed over MiniKube

### DISCUSSION THREAD

**Discussion by we-wake-act21** [eclipse-theia/theia-cloud#290] [Q&A]
Currently, I'm encountering a problem with deploying Theia-cloud on Minikube. I'm unable to open any files, including untitled.txt. However, I can create new files and edit them using the terminal. Here are the steps I've taken so far:

1. Started the pod with a security context configured for the root user.
2. Set the Minikube driver to Docker.

![image](https://github.com/eclipsesource/theia-cloud/assets/116638890/bfe25f55-62bf-4488-8e16-234b0175749a)

I am unable to verify this issue, since no logs are generated when double-clicking on file.

**Comment by jfaltermeier** [comment]
Hi, have you checked both the browser console and the log of the pod for any messages?
Is there anything in the "Network" Tab of the Browser Debugging Tools that looks different than when you run your Theia application from sources?
